.gitignore

```diff
@@ -26,6 +26,7 @@ vscode-extension/**/tsconfig.tsbuildinfo
 vscode-extension/**/package-lock.json
 vscode-extension/**/.eslintcache
 test/dist
+packages/malloy-syntax-highlight/coverage
```

Comment:
You can also create a .gitignore in your package for package specific ignores.

Reply:
Done